minion/consumer_group_offsets.go

```diff
@@ -24,7 +24,7 @@ func (s *Service) ListAllConsumerGroupOffsetsAdminAPI(ctx context.Context) (map[
 	}
 	groupIDs := make([]string, len(groupsRes.AllowedGroups.Groups))
 	for i, group := range groupsRes.AllowedGroups.Groups {
-		if group.GroupState != "Empty" && group.GroupState != "Dead" {
+		if group.GroupState != "Dead" {
```

Comment:
these are Smarts groups right? where members join the group only commiting offsets and else the group remains empty right?

Reply:
yes, Smarts is one example here